playtron-os-scripts/playtron-os-scripts.spec

```diff
@@ -27,6 +29,7 @@ cp playtron-os-scripts-%{ver}/LICENSE %{buildroot}/usr/share/licenses/playtron-o
 
 %files
 /etc/sysctl.d/50-swappiness.conf
+/usr/bin/hwctl
```

Comment:
We need to at least add a `Requires: fio` for the benchmark functionality of `hwctl` (you can append to the existing line so it is: `Requires: cloud-utils-growpart fio`). Anything else?